### ISSUE TITLE
[Usable – Make Code – Home page] Narrator focus is landing on decorative image when navigating in Scan mode under ‘Home page’.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3432]
**User Experience**:
User who depends upon screen reader will be confused if screen reader just mention image and may think that the information inside the image is missing for AT dependent users.

**Note**: User credentials should NOT be included in the bug. 

**Repro Steps:**
1. Launch application URL : https://makecode.microbit.org/# 
2. Home page will open.
3. Navigate to the controls appearing in Home page section.  
4. Observe the narrator behavior in scan mode as focus moves on decorative image or not.

**Actual Result:**
Narrator focus is landing on decorative image when navigating in Scan mode under ‘Home page’ and does not provide any information.

**Expected Result:**
Whenever the user navigating with the narrator in scan mode then the narrator focus does not move to decorative image as Alt =" " is defined for the image available on the ‘Home page’ page.

**Note:**
This issue does not exists on Firefox & Chrome.

**Reference Links:**
•	External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information please use this [Link](https://microsoft.sharepoint.com/teams/ceacc/SitePages/External-Bug-process.aspx). “HCL Staff should not log any third party external bugs. Should be routed to EDAD team”.
•	Please reach out to [C&AI Teams channel](https://teams.microsoft.com/dl/launcher/launcher.html?url=%2F_%23%2Fl%2Fteam%2F19%3A3cd071821b694f11b054d878453cd75f%40thread.skype%2Fconversations%3FgroupId%3D30e3f968-15dd-4da2-bd8e-85f27e4af2c3%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&type=team&deeplinkId=99c73412-3928-459b-a051-ad90788e6969&directDl=true&msLaunch=true&enableMobilePage=true&suppressPrompt=true) for any process related queries.

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0)
Screen Reader: Narrator
#URL:https://makecode.microbit.org/#
